experimental/sdg-static/js/index.js

```diff
@@ -89,18 +89,42 @@ function renderBlocks() {
  * Renders a data commons chart
  */
 function renderChart($container, chartConfig) {
-  if (chartConfig.type == "LINE") {
+  if (chartConfig.type == "BAR") {
+    const chartElement = document.createElement("div");
+    $container.append(chartElement);
+    var lineChartProps = {
+      id: _.uniqueId("chart-"),
+      svgChartHeight: 200,
+      className: undefined,
```

Comment:
what is className used for?

Reply:
this lets users add a custom className (and thus to apply custom styles) to charts. It defaults to undefined, but is overridden if set in the sdg.json config file. I can add an example of this in a follow-up pr